onnx/defs/quantization/defs.cc

```diff
@@ -85,13 +94,14 @@ ONNX_OPERATOR_SET_SCHEMA(
         }));
 
 static const char* DequantizeLinear_ver21_doc = R"DOC(
-The linear dequantization operator. It consumes a quantized tensor, a scale, and a zero point to compute the full precision tensor.
-The dequantization formula is `y = (x - x_zero_point) * x_scale`. `x_scale` and `x_zero_point` must have same shape, and can be either a scalar
-for per-tensor / per layer quantization, or a 1-D tensor for per-axis quantization.
-`x_zero_point` and `x` must have same type. `x` and `y` must have same shape. In the case of dequantizing int32,
+The linear dequantization operator. It consumes a quantized tensor, a scale, and a zero point to compute the full-precision tensor.
+The dequantization formula is `y = (x - x_zero_point) * x_scale`. `x_scale` and `x_zero_point` must have the same shape, determining the quantization's granularity:
+a scalar for per-tensor/per-layer quantization, a 1-D tensor for per-axis quantization, or have a rank identical to the input for blocked quantization.
+Blocked quantization is not supported for 1-D input tensors. See QuantizeLinear for details on quantization granularity.
```

Comment:
What's the reason for blocked quantization not being supported for 1D input tensor?

Reply:
This is related to inferring of the block size. However, I'll support this use case as well, since I'll be adding the `block_size` attribute.

See here for previous discussion: https://github.com/onnx/onnx/pull/5812#discussion_r1440515492

Reply:
Great, that would be helpful.